source/php/Component/Hero/hero.json

```diff
@@ -3,6 +3,7 @@
     "default": {
         "image": false,
         "imageFocus": {"top": 30, "left": 50},
+        "video": false,
```

Comment:
Can be hard to understand what you should put into video and image since the type here is bool. meaning the value can be either bool or string. to streamline the type we could set empty string "" as default. just an idea

Reply:
I inherited the style of default in "image" parameter. These would shurely benefit form att type-definition added to the .json definition. 

Like "types": {image: "bool||string"}
Or a more in-depth validation: "types": {image: "bool||url"}


This may be discussed further.